### ISSUE TITLE
Try flink-crawler with crawler-commons 0.10 release candidate

### ISSUE THREAD

**Issue by kkrugler** [ScaleUnlimited/flink-crawler#154]
From Julien:

> I have pushed a RC for 0.10
>
>The list of changes can be found [here](https://github.com/crawler-commons/crawler-commons/blob/crawler-commons-0.10/CHANGES.txt)
>
>Artifacts => https://oss.sonatype.org/content/repositories/comgithubcrawler-commons-1006
>
>You can test it with Maven by inserting the following into your pom.xml
>
``` xml
<repositories>
    <repository>
      <id>my-repo2</id>
      <name>your custom repo</name>
      <url>https://oss.sonatype.org/content/repositories/comgithubcrawler-commons-1006/</url>
    </repository>
  </repositories>
```

**Comment by vmagotra** [comment]
I have created a branch for this 154_crawlercommons-0.10 and updated the pom to use version 0.10. It seems to be working.

**Comment by vmagotra** [comment]
0.10 has been released and the artifact is available via the Maven central repo, so I have updated the pom to remove the sonatype repo. It is working.

**Comment by kkrugler** [comment]
Merged change to switch to 0.10 into master, so all good.
